pdc/apps/compose/lib.py

```diff
@@ -262,12 +264,52 @@ def compose__import_images(request, release_id, composeinfo, image_manifest):
     return compose_obj.compose_id, imported_images
 
 
+def _set_compose_tree_location(request, compose_id, composeinfo, location, url, scheme):
+    ci = productmd.composeinfo.ComposeInfo()
+    common_hacks.deserialize_wrapper(ci.deserialize, composeinfo)
+    num_set_locations = 0
+    synced_content = [item.name for item in ContentCategory.objects.all()]
```

Comment:
This implementation is mentioned in JIRA. It looks can't get synced_content from inputs.